docs/computing/alliancecan.md

```diff
@@ -57,14 +57,29 @@ into a terminal):
 
 ### Mounting Graham data locally with `sshfs`
 #### Mac/Linux
-Detailed instructions for mounting can be found 
-[here](https://www.sharcnet.ca/help/index.php/SSHFS), however, make sure to use
-the options highlighted below:
+After setting up the passwordless SSH, follow the next steps:
+1. Create a new bash file (location doesn't matter) with the following content:
+    ```bash
+    #!/bin/bash
+    mount_dir="<MOUNT_DIR>"
+    if (! mountpoint -q $mount_dir); then
+        sshfs <user>@graham.sharcnet.ca:/home/<user>/ $mount_dir -o   ServerAliveInterval=15,ServerAliveCountMax=3,Compression=no,follow_symlinks
+    else
+        umount $mount_dir
+    fi
+    ```
+    Replace `<MOUNT_DIR>` with the directory to which you want to mount your Graham home folder. Also, substitute `<user>` with your corresponding user. \
+    The name of the file will be the command that you will use to mount and unmount your Graham folder, so make sure to use an easy to remember; for example, `graham_sshfs.sh`.
+2. Rename the file to remove the `.sh` extension.
+3. Create a new folder in your local home folder: `mkdir -p ~/bin`. 
+4. Copy the previous file to that location (`cp <file_name> ~/bin`) and add execution permissions to it (`chmod 755 ~/bin/<file_name>`).
+5. Add the following line at the end of your bashrc file:
+    ```bash
+    export PATH=$PATH:~/bin
+    ```
+6. Reload your bashrc file: `source ~/.bashrc`.
 
-```bash
-mkdir $HOME/graham;
-sshfs -o reconnect,ServerAliveInterval=15,ServerAliveCountMax=3,follow_symlinks $USER@gra-dtn1.computecanada.ca:/home/$USER ~/graham;
-```
+Now, you should be able to mount or unmount your Graham home folder by running `<file_name>` in the terminal.
```

Comment:
I think my preference here would just be to combine the steps to avoid extra steps and multiple copies of files. Something along the lines of:

1. Create `~/bin` 
2. Create the bash script in that location 
3. Change permissions - maybe include a short note of what `755` is or replace with `+x` instead
4. Export to path.
5. Sourcing `~/.bashrc`